msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/flw/TestCase833513.java

```diff
@@ -117,7 +117,7 @@ public void test_833513() throws MsalException, InterruptedException, LabApiExce
                 .broker(mBroker)
                 .prompt(PromptParameter.SELECT_ACCOUNT)
                 .expectingBrokerAccountChooserActivity(false)
-                .howWouldYouLikeToSignInExpected(true)
+                .howWouldYouLikeToSignInExpected(false)
```

Comment:
interesting

Reply:
I remember having to add this in for adfsv3 tests either last month or the month before, we were getting an extra ui page. But now that seems to be gone...